Arduino_package/package_realtek.com_amebad_early_index.json

```diff
@@ -10,7 +10,43 @@
       },
 
       "platforms": [
-		{
```

Comment:
This update and the related "ameba_d-3.0.8-build20210402.tar.gz" are not necessary and rejected. The official release will approach once this PR is merged.